src/Subscription.php

```diff
@@ -67,6 +74,66 @@ public function owner()
         return $this->belongsTo($model, (new $model)->getForeignKey());
     }
 
+    /**
+     * Get the subscription items related to the subscription.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function items()
+    {
+        return $this->hasMany(SubscriptionItem::class);
+    }
+
+    /**
+     * Determine if the subscription has multiple plans.
+     *
+     * @return bool
+     */
+    public function hasMultiplePlans()
+    {
+        return is_null($this->stripe_plan);
```

Comment:
Shouldn't this be if the count > 0?

Reply:
No, a subscription without a stripe plan set on it has multiple subscriptions.